src/index.js

```diff
@@ -3,7 +3,7 @@
  */
 
 import debugFactory from 'debug';
-import Limiter, { memoryAdapter, redisAdapter } from 'ratelimiter';
+import Limiter, { memoryAdapter, nullAdapter, redisAdapter } from 'ratelimiter';
```

Comment:
node-ratelimiter is not modified in the pr because in our fork, we reference the marmelab fork of node-ratelimiter directly (github ref)